### PR TITLE
Feat: Filter genuine email #21

### PR DESCRIPTION
**Pull request by sk-upadhyay**
## PR Description

Integrated disify api for filtering out disposable email by verify_email
Removed data redundancy by implementing cuid to be assigned to only one person  
changes need to be done in SQL is to make CUID as primary key and remove index as primary key 
## Related Issues: Issue for which you are raising a PR for

- Closes #

### Issue #21 

[Issue number or description]

### Checklist

- [x] I have gone through the [contributing guide](https://github.com/SaiyamTuteja/Faculty_Evaluation_System/blob/main/CONTRIBUTING.md)
- [x] I have updated my branch and synced it with project `main` branch before making this PR
- [x] Is this a bug fix/enhancement/documentation changes
- [x] Part of GSSOC
- [x] Tested for any breaking changes
- [x] Other relevant checks completed

## Undertaking

I declare that: 

1. The content I am submitting is original and has not been plagiarized.  
2. No portion of the work has been copied from any other source without proper attribution.  
3. The work has been checked for plagiarism, and I assure its authenticity.  

I understand that any violation of this undertaking may have legal consequences that I will bear and could result in the withdrawal of any recognition associated with the work. 

- [x] I Agree

### CODE REVIEW

**Comment by sk-upadhyay** [comment]
@aswinikalyan30 Any Update on my PR

**Comment by aswinikalyan30** [comment]
Hey. This has changes that solves an issue not assigned to you

File: login.php [hunk 1]
```diff
@@ -1,134 +1,160 @@
...
-              <input type="checkbox" id="remember">
-              <label for="remember">
-                Remember Me
-              </label>
+          <div class="input-group mb-3">
+            <input type="password" class="form-control" name="password" required placeholder="Password">
+            <div class="input-group-append">
+              <div class="input-group-text">
+                <span class="fas fa-lock" onclick="togglePasswordVisibility()"></span>
+                <span class="fas fa-eye" style="display:none;" onclick="togglePasswordVisibility()"></span>
```
Comment (aswinikalyan30) [inline login.php:70]: The toggleVisibility is already being worked on by someone else

**Review by aswinikalyan30 — APPROVED** [review]
Looks good otherwise, please add a short video/screenshots to the PR so that it is easier to see the changes.

**Comment by sk-upadhyay** [comment]
@aswinikalyan30 so what should i do?? should i update and pr again or i should close it.

**Comment by sk-upadhyay** [comment]
@SaiyamTuteja Can you merge my PR

**Pull request merged**